suite-native/module-trading/src/components/general/TradeableAssetsSheet/FavouriteIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import { Pressable } from 'react-native';
+
+import { Icon, IconColor, IconName } from '@suite-native/icons';
+import { useTranslate } from '@suite-native/intl';
+
+export type FavouriteIconProps = {
+    isFavourite: boolean;
+    onPress: () => void;
+};
+
+export const FavouriteIcon = ({ isFavourite, onPress }: FavouriteIconProps) => {
+    const { translate } = useTranslate();
```

Comment:
maybe using something like for presets:
```const iconData = {
    favourite: {
        name: '',
        color: '',
        hint: '',
    },
    normal: {
        name: '',
        color: '',
        hint: '',
    },
} as const;```

would make it more readable? wdyt?
```suggestion
    const { translate } = useTranslate();
```

Reply:
Updated. I applied a bit different approach. Hope you like it.